xray/audit/jas/jasmanager.go

```diff
@@ -39,6 +41,7 @@ func GetExtendedScanResults(xrayResults []services.ScanResponse, dependencyTrees
 	if err = utils.CreateAnalyzerManagerLogDir(); err != nil {
 		return nil, err
 	}
+	scannersToExclude = strings.Split(excludeScan, ";")
```

Comment:
Let's also trim strings to allow for example: `--exclude-scan=contextual_analysis; secrets`